docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -254,7 +254,7 @@ To add a Semgrep configuration snippet into Bitbucket Pipelines:
 2. Copy the relevant code snippet provided in [Sample Bitbucket Pipelines configuration snippet](#sample-bitbucket-pipelines-configuration-snippet), and then paste it to your `bitbucket-pipelines.yml`.
 3. Commit the updated `bitbucket-pipelines.yml` configuration file.
 4. The Semgrep job starts automatically upon detecting the committed `bitbucket-pipelines.yml` file. You can also view the job through Bitbucket's interface, by clicking **your repository > Pipelines**.
```

Comment:
```suggestion
4. The Semgrep job starts automatically upon detecting the committed `bitbucket-pipelines.yml` file. You can view the job through Bitbucket's interface, by clicking **your repository > Pipelines**.
```
You didn't change this line, but the "also" doesn't make sense, and this seems like a good time to remove it.